### PR TITLE
Text clean-up

### PR DESCRIPTION
**Pull request by deanna-minnick**
## Description & motivation
This PR removes whitespace from an inline code snippet and removes an "Options" header that had no contents.

### CODE REVIEW

File: website/docs/reference/commands/compile.md [hunk 1]
```diff
@@ -9,8 +9,6 @@ The `compile` command is useful for:
 
 1. Visually inspecting the compiled output of model files. This is useful for validating complex jinja logic or macro usage.
 2. Manually running compiled SQL. While debugging a model or schema test, it's often useful to execute the underlying `select` statement to find the source of the bug.
-3. Compiling `analysis `files. Read more about analysis files [here](analyses).
+3. Compiling `analysis` files. Read more about analysis files [here](analyses).
```
Comment (mirnawong1) [inline website/docs/reference/commands/compile.md]: adding link of full file path

```suggestion
3. Compiling `analysis` files. Read more about analysis files [here](/docs/build/analyses).
```
```suggestion
3. Compiling `analysis` files. Read more about analysis files [here](analyses).
```

**Review by mirnawong1 — DISMISSED** [review]
thanks again @deanna-minnick - one small tweak!

**Comment by deanna-minnick** [comment]
> thanks again @deanna-minnick - one small tweak!

good catch, @mirnawong1! I've committed that change and merged the most recent changes from `current` into this branch.

**Review by mirnawong1 — APPROVED** [review]
Approved!

**Pull request merged**